cli-definition.json

```diff
@@ -141,16 +141,6 @@
     }
   ],
   "flags": [
-    {
```

Comment:
I'd have expected these flags to have moved, not just removed.

I wonder if the tool that generates this CLI description is working correctly.